contracts/src/types/FulfillmentContext.sol

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2025 RISC Zero, Inc.
+//
+// All rights reserved.
+pragma solidity ^0.8.24;
+
+using FulfillmentContextLibrary for FulfillmentContext global;
+
+/// @title FulfillmentContext
+/// @notice A struct for storing validated fulfillment information in transient storage
+/// @dev This struct is designed to be packed into a single uint256 for efficient transient storage
+struct FulfillmentContext {
+    /// @notice Boolean set to true to indicate the request was validated
+    bool valid;
+    /// @notice The validated price for the request
+    uint96 price;
+    /// @notice The address to call during fulfillment. If set to address(0), no callback is performed
+    address callback;
+    /// @notice The gas limit to use for the callback. Must be non-zero if callback is set
+    uint96 callbackGaslimit;
```

Comment:
If we put the callback info into the assessor journal, we shouldn't need this here.